spec/std/concurrent_spec.cr

```diff
@@ -28,7 +28,7 @@ describe "concurrent" do
 
     it "re-raises errors from Fibers as ConcurrentExecutionException" do
       exception = expect_raises(ConcurrentExecutionException) do
-        a, b = parallel(raising_job, raising_job)
```

Comment:
This is not useless per-se. It does matter that the return type is a tuple. But it is tested in the following spec.

Reply:
So, this spec is not good then. `parallel(raising_job, raising_job)` raises an error and never assigns to a, b. If you are talking about compile-time checking, check out the test above.